Reflection/Ground.simba

```diff
@@ -10,21 +10,52 @@ Example:
 *)
 Function R_GetMainscreenGroundItemByID(ItemID: Int32): Array of RSTile;
 var
-  I, W, H: Integer;
+  I: Integer;
   ResultTile: RSTile;
   Items: Array of RSTile;
+  MSBounds: TBox;
 begin
   Items := RSGroundObject.GetTileByID(ItemID);
-  if Length(Items) < 1 then
+  MSBounds := R_MainscreenBounds;
+  if High(Items) < 1 then
     Exit;
```

Comment:
You could invert the order of these calls to sometimes save a call

Reply:
Fixed! Thanks for that find!